### PR TITLE
Define branch of container-index to look for projects

### PR DESCRIPTION
**Pull request by navidshaikh**
Allow specifying branch of container index in inventory file. With this user can specify the branch of container-index to look for projects to be built.

### CODE REVIEW

**Comment by navidshaikh** [comment]
With this in place, we can have one container-index with different branches viz ci, pre-prod, prod-ready, etc.

**Comment by bamachrn** [comment]
this looks good to me.

File: ci/lib.py [hunk 1]
```diff
@@ -166,7 +166,8 @@ def generate_ansible_inventory(jenkins_master_host, jenkins_slave_host,
 log_level=DEBUG
 openshift_server_ip={openshift_host}
 deployment=ci
-cccp_index_repo=https://github.com/rtnpro/container-index.git
+cccp_index_repo=https://github.com/navidshaikh/cccp-index.git
```
Comment (bamachrn) [inline ci/lib.py]: I would say lets make it `https://github.com/centos/container-index.git` with CI branch

Comment (navidshaikh) [reply ci/lib.py]: that was purposefully added to have the PR tested in CI if it works with a git repo and a branch, will replace the repo URL in another commit.

**Comment by navidshaikh** [comment]
@bamachrn : Please create a branch `ci` for container-index repository to raise PR against.

**Comment by navidshaikh** [comment]
The next update to this PR (about pointing to some existing git branch i.e. `ci` for production container-index repository), will be meaningful only after https://github.com/CentOS/container-index/pull/228 merges.
Since, updating now will waste on CI cycle. (as CI won't get the mentioned branch untill PR is merged)

**Comment by navidshaikh** [comment]
@bamachrn : This is ready to merge.

**Comment by navidshaikh** [comment]
@dharmit @bamachrn @mohammedzee1000 : This is ready to merge.

**Pull request merged**